examples/azure/README.md

```diff
@@ -0,0 +1,32 @@
+# Provision an Azure cluster
+
+## Overview
+
+This example shows you how to use Hydroform to provision an Azure cluster.
+
+## Installation
+
+### Configure Azure
+
+To provision a Azure cluster you need the client ID and client secret of your azure subscription stored in a TOML file as follows:
+
+```toml
+CLIENT_ID = "<my-client-id>"
+CLIENT_SECRET = "<my-client-secret>"
+```
+
+### Run the example
+
+1. To provision a new cluster on Azure, go to the `hydroform` directory and run:
+
+```bash
+go run ./examples/azure/main.go -p {resource_group_name} -c /{path/to/credentials.toml}
+```
+
+2. On the Azure portal, go to **Home** > **All Resources** to see your cluster on the list.
+
+4. Export **KUBECONFIG** environment variable pointing to the `kubeconfig` file generated by running the example. This will allow access to the cluster.
```

Comment:
```suggestion
3. Export the **KUBECONFIG** environment variable pointing to the `kubeconfig` file generated by running the example. This will allow you to access the cluster.
```